samples/files.js

```diff
@@ -116,6 +116,8 @@ async function uploadFile(bucketName, filename) {
   await storage.bucket(bucketName).upload(filename, {
     // Support for HTTP requests made with `Accept-Encoding: gzip`
     gzip: true,
+    // By setting the option `destination`, you can change the name of the
```

Comment:
It's a little weird for me to read the comment in this context.  I'd expect to see something like this above a `destination` property in the obj, which could be commented out, but there for illustration.

Reply:
so perhaps:

`// destination: destinationFilename (defaults to filename)`